flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/scala/expressionDsl.scala

```diff
@@ -691,6 +692,8 @@ trait ImplicitExpressionConversions {
   implicit def sqlTimestamp2Literal(sqlTimestamp: Timestamp): Expression =
     Literal(sqlTimestamp)
   implicit def array2ArrayConstructor(array: Array[_]): Expression = convertArray(array)
+  implicit def UserDefinedAggFunctionConstructor[T: TypeInformation, ACC](udagg:
+  AggregateFunction[T, ACC]): UDAGGExpression[T, ACC] = UDAGGExpression(udagg)
```

Comment:
Do we need this `UDAGGExpression`? Can't we convert directly into `UDAGGFunctionCall`? Imho I would call this `AggregateFunctionCall` and avoid acronyms.